packages/salesforcedx-vscode-lwc/src/util/workspaceUtils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { workspace } from 'vscode';
+
+export class WorkspaceUtils {
+  private context: vscode.ExtensionContext | undefined;
+  private static instance: WorkspaceUtils;
+
+  private constructor() {}
+
+  public static getInstance(): WorkspaceUtils {
+    if (!this.instance) {
+      this.instance = new WorkspaceUtils();
+    }
+    return this.instance;
+  }
+
+  public init(extensionContext: vscode.ExtensionContext) {
+    this.context = extensionContext;
+  }
+
+  public getGlobalStore(): vscode.Memento | undefined {
+    if (this.context === undefined) {
+      return undefined;
+    }
+
+    return this.context.globalState;
```

Comment:
```suggestion
    return this.context && this.context.globalState
```

Reply:
👍  Pretty new to Typescript and I think it shows.  Thanks.